subsys/net/ip/net_shell.c

```diff
@@ -1413,7 +1413,8 @@ static void print_dns_info(struct dns_resolve_context *ctx)
 
 	printk("DNS servers:\n");
 
-	for (i = 0; i < CONFIG_DNS_RESOLVER_MAX_SERVERS + MDNS_SERVER_COUNT; i++) {
+	for (i = 0; i < CONFIG_DNS_RESOLVER_MAX_SERVERS +
```

Comment:
@tbursztyka, @jukkar: Re: this comment: https://github.com/zephyrproject-rtos/zephyr/pull/7251/files#r198456907

That's where it gets a bit hairy, yeah.

Reply:
Hmm, I missed this change. Will fix.